### DISCUSSION TITLE
Disable email verification for specific OAuth Provider

### DISCUSSION THREAD

**Discussion by Schurl31** [fief-dev/fief#303] [Q&A]
Hi, I want to ask if it will be possible to disable email verification for some OAuth Providers?
In my case I have a application whitch allow self registration for any user, but members of my company should sign in without email verification.

**Comment by fief-bailiff** [comment]
Hail, @Schurl31 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
Hi @Schurl31 👋 

Currently, it's not possible to disable email verification, but we are working on a option to disable it per tenant. However, it won't be possible to disable it for a specific OAuth Provider.

**Comment by ScottFred** [comment]
@Schurl31 [The Fief Roadmap](https://docs.fief.dev/#project-roadmap) looks to provide support for a "User Invitation Process".  So, for Enterprise users such as the members of your company, you could ensure they get an invitation sent from you as the Admin.  You could then avoid the Email Verification when you create their accounts... Would that be a workaround?

**Reply by frankie567** [reply]
It could be, but I fear we're still a long way from User's Invitation 😅
